### ISSUE TITLE
glitch adding sources to topic

### ISSUE THREAD

**Issue by hroberts** [mediacloud/web-tools#1479]
1 - go to create new topic screen
2 - click on 'add media' button
3 - add the following collections from the 'featured & starred' tab:

```
U.S. Top Online News 2017
U.S. Top Digital Native News 2016
Left
Center Left
Center
```

4 - click on the 'search geographic collections' tab
5 - enter 'united states' into the search box
6 - try to click on either of the first two results

When I do this, the first set of featured collections works fine, but I get no response when I try to add the US geographic collections.  At some point after clicking around for a bit, I was able to get the US National collection button to work, but still not the US Local / Regional collection.  I eventually had to star the US Local / Regional collection and add it from that screen (which worked fine).  When reproducing to create this issue, I was able to add the US geographic collections fine if I added them before the featured collections.

**Comment by cindyloo** [comment]
this is likely fixed due to the media-picker updates. will confirm

**Comment by dsjen** [comment]
@cindyloo -- did this end up being fixed? Perhaps this will be addressed in the refactor, #1875.
